### PR TITLE
Tame those translations - throw an error if a text string isn't defined with description

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFLSBP-245

#### Description
This PR solves a problem we are having constantly - when dpl-react and dpl-cms isn't aligned properly and translatable strings don't match between the repositories. Now whenever that happens we now throw an descriptive error saying what translation string is missing a definition.

#### Screenshot of the result
n/a

#### Additional comments or questions
n/a

### CODE REVIEW

File: src/components/blocked-patron/blocked-modal/BlockedModal.tsx [hunk 1]
```diff
@@ -17,6 +17,10 @@ const BlockedModal: FC<BlockedModalProps> = ({ blockedStatus }) => {
 
   const { blockedModal } = getModalIds();
 
+  if (!blockedStatus || blockedStatus === "") {
+    return null;
+  }
```
Comment (spaceo) [inline src/components/blocked-patron/blocked-modal/BlockedModal.tsx:22]: This is coming out of the blue :) 
Could you please write a comment why it is necessary?

**Review by spaceo — COMMENTED** [review]
Great work! Nice to have the strings tidyed up!
A have a few questions, apart from that it looks fine.

File: src/components/find-on-shelf/FindOnShelfModal.dev.tsx [hunk 2]
```diff
@@ -12,15 +12,24 @@ import {
...
   mockedPeriodicalManifestationData
 } from "./mocked-data";
+import globalTextArgs from "../../core/storybook/globalTextArgs";
+
+const WrappedFindOnShelfModal = withText(
+  withUrls(withConfig(FindOnShelfModal))
+);
+const WrappedMaterialButtonsFindOnShelf = withText(
+  withUrls(withConfig(MaterialButtonsFindOnShelf))
+);
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/components/find-on-shelf/FindOnShelfModal.dev.tsx:22]: Nopes, because the translations were not being translated otherwise.

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**